.github/workflows/buildtest.yml

```diff
@@ -33,6 +33,12 @@ jobs:
         if: ${{ matrix.goarch }} == "amd64"
         run: sudo make test-race # sudo needed for netns change in test
 
+      - name: Integration test for ${{ matrix.goarch }}
```

Comment:
do we need to run on other arch than amd64 ?

Reply:
as long as it doesn't create problems, we can run them on all architectures